<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scan</class>
 <widget class="QWidget" name="Scan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="timePort">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>0.990000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sélectionner le timeout des ports:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="checkBoxArp">
         <property name="text">
          <string>ARP Ping</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="checkBoxIcmp">
         <property name="text">
          <string>ICMP Ping</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="timeArp">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sélectionner le nombre de paquet pour les pings:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sélectionné votre carte réseau:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sélectionner le temps d'attente des paquets:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quel type de ping souhaitez vous:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listInterface"/>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="timeIcmp">
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonScan">
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QListWidget" name="listUsers"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxIcmp</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxIcmp</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeArp</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxArp</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxArp</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeIcmp</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
